kong/include/kong/services/legal/README.md

```diff
@@ -0,0 +1,16 @@
+# Legal service
+
```

Comment:
Please remove this. This was not supposed to be part of OSS.

Reply:
done